packages/markdown/loader.js

```diff
@@ -157,7 +160,7 @@ module.exports = async function demoLoader() {
       // The import paths currently use a completely different format.
       // They should just use relative imports.
       let moduleID = `./${demoName.replace(
-        `pages/${pageFilename.replace(/^\/src\/pages\//, '')}/`,
+        `pages/${fileRelativeContext.replace(/^docs\/src\/pages\//, '')}/`,
```

Comment:
Mostly stylistic, but this line feels a bit strange. Isn't

```ts
`pages/${fileRelativeContext.replace(/^docs\/src\/pages\//, '')}/`
```
the same as
```ts
`${fileRelativeContext.replace(/^docs\/src\//, '')}/`
```
or even
```ts
`${path.posix.relative('docs/src', fileRelativeContext)}/`
```
?